documentation/src/docs/table-api.md

```diff
@@ -0,0 +1,144 @@
+|`withCheckpointInitiateTimeout`|The timeout for executing a checkpoint of the Reader Group state.  _Applies only to streaming API._|
+|`withSchema`|The table schema which describes which JSON fields to expect.|
+|`withProctimeAttribute`|The name of the processing time attribute in the supplied table schema.|
+|`withRowTimeAttribute`|supply the name of the rowtime attribute in the table schema, a TimeStampExtractor instance to extract the rowtime attribute value from the event and a `WaterMarkStratergy` to generate watermarks for the rowtime attribute.|
+|`failOnMissingField`|A flag indicating whether to fail if a JSON field is missing.|
+
+### Custom Formats
+To work with stream events in a format other than JSON, extend `FlinkPravegaTableSource`. Please see the implementation of [`FlinkPravegaJsonTableSource`](https://github.com/pravega/flink-connectors/blob/master/src/main/java/io/pravega/connectors/flink/FlinkPravegaJsonTableSource.java) for more details.
+
+### Time Attribute Support
+With the use of `withProctimeAttribute` or `withRowTimeAttribute` builder method, one could supply the time attribute information of the event. The configured field must be present in the table schema and of type `Types.SQL_TIMESTAMP()`.
+
+## Table Sink
+A Pravega Stream may be used as an append-only table within a Flink table program.  The Flink Table API is oriented around Flink's `TableSchema` classes which describe the table fields.  A concrete subclass of `FlinkPravegaTableSink` is then used to write table rows to a Pravega Stream in a particular format. The connector library provides out-of-box support for JSON-formatted data with `FlinkPravegaJsonTableSource`, and may be extended to support other formats.
+
+#### Example
+
+The following example uses the provided table sink to write JSON-formatted events to a Pravega Stream:
+
+```java
+// Create a Flink Table environment
+StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+StreamTableEnvironment tableEnv = TableEnvironment.getTableEnvironment(env);
+
+// Load the Pravega configuration
+PravegaConfig config = PravegaConfig.fromParams(ParameterTool.fromArgs(args));
+
+// Define a table (see Flink documentation)
+Table table = ...
+
+// Write the table to a Pravega Stream
+FlinkPravegaJsonTableSink sink = FlinkPravegaJsonTableSink.builder()
+    .forStream("sensor_stream")
+    .withPravegaConfig(config)
+    .withRoutingKeyField("sensor_id")
+    .withWriterMode(EXACTLY_ONCE)
+    .build();
+table.writeToSink(sink);
+```
+
+### Parameters
+A builder API is provided to construct an instance of `FlinkPravegaJsonTableSink`.  See the table below for a summary of builder properties.  Note that the builder accepts an instance of `PravegaConfig` for common configuration properties.  See the [configurations](configurations.md) wiki page for more information.

```

Comment:
Markdown page instead of wiki page to avoid confusion?

Reply:
fixed